randovania/games/dread/logic_database/Dairon.txt

```diff
@@ -540,19 +540,19 @@ Extra - asset_id: collision_camera_008
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
   > Door to Power Switch 1
-      After Dairon - Power Switch 1
+      Trivial
   > Door to Wide Beam Room
       After Dairon - Power Switch 1
   > Tunnel to Power Switch 1
-      Trivial
+      Can Slide
```

Comment:
As you said, this isn't actually needed since the dock weakness already covers it. It all looks fine though.